### PR TITLE
Chapter 4

### PR DESCRIPTION
**Pull request by longlb-0945**
Screenshot from 2019-09-04 19-41-46

### CODE REVIEW

**Comment by longlb-0945** [comment]
ready

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title><%= yield(:title) + " | " + t(".subtitle") %></title>
+    <title><%= full_title(yield :title) %></title>
```
Comment (phucmh-1356) [inline app/views/layouts/application.html.erb]:
```suggestion
    <title><%= full_title yield(:title) %></title>
```

**Comment by longlb-0945** [comment]
ready

**Comment by phucmh-1356** [comment]
merged

**Pull request merged**